clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1661,6 +1663,10 @@ public void handleResponse(AbstractResponse response) {
                         || error == Errors.COORDINATOR_LOAD_IN_PROGRESS) {
                     // If the topic is unknown or the coordinator is loading, retry with the current coordinator
                     continue;
+                } else if (error == Errors.INVALID_PRODUCER_EPOCH
```

Comment:
what was the reasoning for the ordering change here? We've moved these errors up, but left the auth and unsupported message format errors lower.

Reply:
Hmm, not sure. There shouldn't be any functional difference, so I moved it back down.